src/components/Modal/Modal.js

```diff
@@ -72,6 +72,15 @@ export const Modal = ({
   isOpen = false,
 }) => {
   const classes = isOpen ? styles.ModalWrapperActive : styles.ModalWrapper
+  const [backDropClass, setBackDropClass] = useState(styles.ModalBackdrop)
+
+  useEffect(() => {
+    setTimeout(() => {
+      setBackDropClass(
+        isOpen ? styles.ModalBackdropActive : styles.ModalBackdrop
+      )
+    }, 1000)
+  })
```

Comment:
Add empty dependency array?

Reply:
Adding empty dependency array here actually makes `useEffect` not working for some reason.

Reply:
I'm not sure why I suggested that. I'm realizing now this probably needs to run every render not just on the first one.